Feature/dev-Jesse (backend/Model)

Jessemccallums opened the pull request:
In this PR, I am creating models for the categories of liquor and wine.

Jessemccallums committed:
Feature/dev-Jesse (backend/Model)  Categories Models for Liquor and Wine was created on this commit

Fer-Mig-Agus reviewed (commented):
Guiate de los modelos que ya estan hechos

Jessemccallums committed:
Solving issues with Models

In this commit I am solving issues with the Models.

The approach was adding unique value to the name property, and also make the ID property in UUID

Fer-Mig-Agus reviewed (commented):
Separar con el guion bajo

Jessemccallums committed:
Update models

In this commit I am updating the Name of the Models, with the porpuse of replace the names with a proper name to use

Fer-Mig-Agus reviewed (approved):
Muy buen trabajo
